.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
```

Comment:
90% of the time, fallthrough in switch is a bug. If you want it, comment so in the specific instance and add an 'ignore linting' line